edgedb/blocking_con.py

```diff
@@ -29,7 +29,7 @@
 
 from .datatypes import datatypes
 from .protocol import blocking_proto
-from .protocol import protocol
+from .protocgitol import protocol
```

Comment:
Looks like an unintended change here.

Reply:
I was talking about Yury's commits in my PR, if it's fine I'll just fix this change